baybe/surrogates/base.py

```diff
@@ -83,6 +86,34 @@ def to_botorch(self) -> Model:
 
         return AdapterModel(self)
 
+    @staticmethod
+    def _get_parameter_scaler(
+        parameter: Parameter,
+    ) -> ScalerProtocol | Literal["passthrough"]:  # noqa: F821
+        """Return the scaling method to be used for the given parameter."""
+        return MinMaxScaler()
```

Comment:
@Scienfitz, what should we actually set as default here? I can't yet decide between `MinMaxScaler` and `"passthrough"` ... 🤔 

`MinMaxScaling` probably is the most sensible default choice, but currently we would pretty much deactivate it for all our surrogates except the linear one. Is that OK for you?

Reply:
wait arent we use Standardize as default currently? Why is minmax preferred to that?

Reply:
Yes, we've been using it. But actually I think minmax makes much more sense, tbh. Also backed up by the fact that botorch uses minmax for input and standardize for output. Seems more intuitive to me. What would be your justification for standardize? 

(And again: effectively, the scaling is relevant only for the linear surrogate at the moment. All other surrogates to their own stuff anyway)

Reply:
ok its prob safest to use botorch defaults too, overall i have the feeling that removing the mean is safer but it doesnt seem necessary

Reply:
Agree here, let's do what botorch does.

Reply:
Done, using minmax scaler by default